internal/source/code.go

```diff
@@ -111,7 +120,46 @@ func StripComments(sql string) (string, []string, error) {
 			comments = append(comments, t)
 			continue
 		}
+		if strings.HasPrefix(t, "#") {
+			comments = append(comments, strings.TrimPrefix(t, "#"))
+			continue
+		}
```

Comment:
This (along with similar change above) was a bug, fixing resulted in one updated mysql test output.